pandas/tests/io/test_sql.py

```diff
@@ -1833,7 +1834,7 @@ def test_read_procedure(self):
         try:
             r1 = connection.execute(proc)  # noqa
             trans.commit()
-        except:
+        except pymysql.Error:
```

Comment:
Looks that the other two tests import `pymysql` locally. This is the only one that should have it (also locally, as in some builds `pymysql` is not installed, and travis is failing

Reply:
Does this mean I should have it imported right before the try statement like I had it imported before? I did that previously because that's how it was done throughout the file.

Reply:
Sorry if I made you change that, I didn't see it was an optional dependency. Yes, at the beginning of the function.